Assets/HoloToolkit/Input/Scripts/Interactable.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Base class Interactable GameObjects can inherit from
```

Comment:
I thought about expanding this a bit, but I wanted to discuss this a bit more with the community.